tests/init/pos/case-class-apply.scala

```diff
@@ -0,0 +1,8 @@
+case class Case(b: Base)
```

Comment:
This tests the `Hot` case in the `call` method but not `Ref`. How do I add such a case?